src/main/java/frc/robot/Robot.java

```diff
@@ -46,13 +51,14 @@
     final private NetworkTableEntry nameEntry = NetworkTableInstance.getDefault().getEntry("RobotName");
     final private String robotName = nameEntry.getString("Unknown");
 
-    final private RobotMap map = RobotUtils.getMapForName(robotName, RobotMap.class, "frc.robot.maps", new RobotMap());
+    final private RobotMap map = new TempestMap();
```

Comment:
You'll need to pull in the changes that @msoucy and @jeffaburke  made to this mechanism and this line should be reverted.